flask_project/campaign_manager/models/models.py

```diff
@@ -0,0 +1,298 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+import os
+
+from sqlalchemy import (
+    create_engine,
+    Column,
+    Integer,
+    String,
+    Table,
+    Text,
+    ForeignKey,
+    Date,
+    DateTime,
+    BOOLEAN
+)
+from sqlalchemy.ext.declarative import declarative_base
+from sqlalchemy.orm import relationship
+from geoalchemy2 import Geometry
+
+# change the db-name, db-owner and db-password
+
+db_location = os.environ['DATABASE_URL']
+engine = create_engine(db_location, echo=True)
+
+Base = declarative_base()
+
+adminAssociations = Table('adminAssociations', Base.metadata,
```

Comment:
pull left

Reply:
The port_database branch is has the correct code format. This branch will be rebased on top of it and it will handle the pull left.